versions.properties

```diff
@@ -1,5 +1,5 @@
 # Scala version used for bootstrapping (see README.md)
-starr.version=2.13.0-RC1
+starr.version=2.13.0-RC2
```

Comment:
I don't know if this is right to change.  (Seems definitely out of scope.)

Reply:
Yes, now is the right time to bump starr. Usually we do this in a separate commit, but nbd.